src/Sulu/Component/Rest/FlattenExceptionNormalizer.php

```diff
@@ -36,6 +43,26 @@ public function normalize($exception, $format = null, array $context = [])
             $data['code'] = $exception->getCode();
         }
 
+        if (\in_array($this->environment, ['dev', 'test'])) {
```

Comment:
Shouldn't we check the `debug` flag instead? Had problems with checks like this recently for the `CachingTest`.

Reply:
Good catch 👍  did update it.